package.json

```diff
@@ -106,5 +106,6 @@
       "core-js",
       "level"
     ]
-  }
+  },
+  "packageManager": "pnpm"
```

Comment:
I don't see such example in the Node.js docs. The Node.js docs only showcase `"<pkgName>@<exactVersion>"`